### PR TITLE
Keadm: optimize reset detect logic

### LINKED ISSUE #2464: When cloudcore not running，keadm reset failed
**What happened**:
I have wrong certs，so I wanna use `keadm reset` to reset node，but it failed and have some errors when check `edge`

code located at `keadm/cmd/keadm/app/cmd/util/common.go:411` 
image

**What you expected to happen**:
skip error，just clean what exist

**How to reproduce it (as minimally and precisely as possible)**:
On cloudcore，`pkill cloudcore` and then run `keadm reset`

**Anything else we need to know?**:
I think we need update cloudcore to running as `systemd service` or when check command failed，just return `false，nil`

**Environment**:

<details><summary>CloudCore Version</summary>

```console
$ cloudcore --version

KubeEdge v1.5.0
```

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:

To detect which module running, no need to check it run as a service or not, so simplify it.

**Which issue(s) this PR fixes**:

Fixes #2464

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/reset.go [hunk 1]
```diff
@@ -67,15 +67,16 @@ func NewKubeEdgeReset(out io.Writer, reset *common.ResetOptions) *cobra.Command
...
 			if err != nil {
 				return err
 			}
 			switch whoRunning {
 			case common.KubeEdgeEdgeRunning:
 				IsEdgeNode = true
 			case common.NoneRunning:
-				return fmt.Errorf("None of KubeEdge components are running in this host")
+				fmt.Println("None of KubeEdge components are running in this host, exit")
+				os.Exit(0)
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/reset.go:79]: Has any difference with `return fmt.Errorf`?

**Review by fisherxu — COMMENTED** [review]
Looks good, but how does this PR fixed the issue linked?

File: keadm/cmd/keadm/app/cmd/reset.go (hunk 1 above)
Comment (daixiang0) [reply keadm/cmd/keadm/app/cmd/reset.go:79]: If return err, `keadm` would print debug stack and help info which becomes confused for users.

Old logic would detect service exists or not, in fact we just consider binary running or not here, so simplify logic would fix the issue.

**Pull request merged**